CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# dbt_netsuite_source v0.10.0
+
+This is part of a batch release. 
+
+## 🎉 Feature Updates 🎉
+- Fields added to support account hierarchies in the `dbt_netsuite` end model. [PR #45](https://github.com/fivetran/dbt_netsuite_source/pull/45)
+  - Added `is_elimination` in `stg_netsuite2__subsidiaries`.
+  - Added `is_eliminate` and `net_amount` in `stg_netsuite2__transaction_lines`.
+
+## Documentation Updates
+- Added `display_name` in `stg_netsuite2__accounts`; added `exchange_rate` in `stg_netsuite2__transaction_accounting_lines`. [PR #45](https://github.com/fivetran/dbt_netsuite_source/pull/45)
```

Comment:
Why are some of the new fields listed under feature updates and some listed under documentation updates? Can we instead list them all as feature updates?

Reply:
Additionally, we should call out for the new `accounts`, `subsidiaries`, and `transaction_lines` fields that if they are currently being passed through in the respective variables, that they will need to be removed. Otherwise they may see an error.

Reply:
Updates applied.